optuna_dashboard/_app.py

```diff
@@ -249,19 +249,17 @@ def create_study() -> BottleViewReturn:
             return {"reason": "You need to set study_name and direction"}
 
         try:
-            study_id = storage.create_new_study(study_name)
+            study_id = storage.create_new_study(
+                study_name,
+                directions=[
+                    StudyDirection.MAXIMIZE if d.lower() == "maximize" else StudyDirection.MINIMIZE
+                    for d in directions
+                ],
+            )
```

Comment:
To support multiple Optuna versions, Optuna Dashboard checks `optuna.__version__` and then calls corresponding storage apis like the following.

https://github.com/optuna/optuna-dashboard/blob/c30fe8234f7ddb2e3206d6fa295a0d174dc9f15d/optuna_dashboard/_app.py#L141-L148

Could you update the implementation to keep backward compatibilities?

Reply:
OK, I see.